README.md

```diff
@@ -52,3 +52,34 @@ If no [token](https://docs.github.com/en/github/authenticating-to-github/creatin
 is provided then you may run into restrictions on the
 amount of times you can update the list. To provide the token define the environment
 variable `GITHUB_TOKEN` with the value of the token.
+
+### Adding companies
+Companies display their logos on the logo wall. To add a new company a new entry
+in `_data/companies.yml` is needed. Each company needs an unique hash to be
+referenced. Place the logo under `assets/img/companies`. The needed data to add
+a company is:
+
+- `name`: Display name of the company
+- `logo`: Path to the logo, relative to `assets/img/companies/`.
+- `url`: URL to the company website
+
+### Adding Use Cases
+Use cases are displayed as cards in the Users section. They also have their own
+page that describes in detail the use case. Use cases are a collection. To add a
+new one, create a new file in the `_use_cases` directory. The first lines will
+contain the metadata of the use case in YAML format. This section must have
+lines before and after containing `---`. Following the metadata, place the main
+content in Markdown format.
+
+Use cases will contain a quote. You will need a photo of the author of the
+quote, place it under `assets/img/use-cases/user-photos`. Each use case must be
+associated to a company (see #adding-companies).
+
+The metadata required for an use case is the following:
+
+- `project`: Project title (used as title of the page)
+- `quote`: Quote from the user
+- `user`: Author of the quote
+- `user_position`: Position of the user in the company
+- `user_photo`: Path to the user photo, relative to `assets/img/use-cases/`
+- `company`: hash of the company in `companies.yml` (see #adding-companies)

```

Comment:
you mean "key" instead of "hash"?

Reply:
Yeah, I just used that term because it was mentioned in the Jekyll documentation (https://jekyllrb.com/docs/datafiles/#example-accessing-a-specific-author). I'll change it to key.

Reply:
Done